### PR TITLE
Add contact form and footer

### PR DESCRIPTION
**Pull request by sumon766**
Add contact form
Add HTML validation with form inputs
Add hover to buttons
Add footer icons

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -275,5 +275,48 @@ <h3>Skills</h3>
...
         <!-- About -->
+
+        <!-- Contact -->
+        <section>
+            <div class="contact">
+                <div class="contact-text">
+                    <h4>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h4>
+                </div>
+                <div class="contact-form">
+                    <form action="" method="post">
```
Comment (christianbiring1) [inline index.html:286]: - [ ] Kindly make sure the form spree service is used correctly. You are supposed to put your own `form_id`. In that way, you will be able to get the user inputs in your mail.
After creating an account on formspree service, just follow the steps listed in the link I just forward down below :smiley: 
`Hint: You may consider checking` [here](https://formspree.io/html/#:~:text=your%20website%27s%20CSS-,How%20it%20works,-With%20a%20couple) how to use the form spree service.

File: index.html [hunk 2]
```diff
@@ -275,5 +275,48 @@ <h3>Skills</h3>
...
+                            <input type="text" name="fName" placeholder="First Name" maxlength="15">
+                            <input type="text" name="lName" placeholder="Last Name" maxlength="15">
+                            <input type="text" name="fullName" placeholder="Full Name" maxlength="30">
+                        </div>
+                        <div id="email">
+                            <input type="email" name="email" required placeholder="Email">
+                        </div>
+                        <div id="message">
+                            <textarea name="message" id="large" placeholder="Hello Mariane, I would like to get in touch with you..." maxlength="500"></textarea>
+                            <textarea name="message" id="small" required placeholder="Enter text here" maxlength="500"></textarea>
```
Comment (christianbiring1) [inline index.html:297]: - [ ] Kindly make sure all your `inputs` fields have the attribute `required` to prompt the user not to submit an empty form as required in the requirements.

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @sumon766 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# STATUS: REQUEST CHANGES :recycle: 
163947610-e4d849d5-e69c-49b9-90bd-a4191f4e2856


## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Descriptive PR `title` and `summary` :+1: 
- [ ] Good looking UI :art: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 3]
```diff
@@ -666,3 +728,134 @@
...
+  background-color: #f55800;
+  padding: 10px 12px;
+  border: 0;
+  cursor: pointer;
+}
+
+.contact-form button:hover,
+.contact-form-mob button:hover {
+  color: #d6c9c9;
+}
```
Comment (alexpuente0) [inline style.css:843]: - [OPTIONAL] Please re-check your design, as location of button and effects are not working properly.
Here are the button effects: 
image

And here's how the submit button needs to be centered:
image

**Review by alexpuente0 — APPROVED** [review]
Hi @sumon766 , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Form is working properly
- Code is well organized

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**